.docs/test_plan.md

```diff
@@ -0,0 +1,157 @@
+ # Test plan of Armonik "2.12.0"
+
+-[Introduction](#Introduction)
+
+-[The product and the functionalities to test](#the-product-and-the-functionalities-to-test)
+
+-[Prerequisite and exigences](#prerequisite-and-exigences)
+
+-[The tools used to do the tests](#the-tools-used-to-do-the-tests)
+
+-[Tests environment](#Tests-environment)
+
+-[Exploited ressources](#exploited-ressources)
+
+-[The estimated results](#the-estimated-results)
+
+-[Tests to do](#tests-to-do)
+
+# Introduction
+
+This test plan describes the performance tests of ArmoniK. The purpose of those tests is to mesure the performance of the differents functionalities of ArmoniK (Tasks submission, Tasks treatment, ...). For this, we will do different tests:
+- Measure the execution time of different numbers of submitted tasks and different size of the tasks.
+- Test the strong scalability.
+
+The purpose of those tests is to have a reference numbers and performance of the versions of ArmoniK.
+
+# The product and the functionalities to test
+
+## Products
+
+- Bench "0.11.4" , Stresstest "2.12.0" on ArmoniK "2.12.0"
+- The submission of the tasks
+- The processing of the tasks
+- The retrieving of the results 
+
+# Prerequisite and exigences
+
+- Deploy ArmoniK with different partitions of bench (different number of pods in each partition)
+
+# The tools used to do the tests
+
+## Scripts bash
+
+- To lunch a warming up run then a bunch of runs with the same parameters and store them in files
+
+## Python
+
+- Clean the files where we stocked the data in readable json files.
+- Read the data and calculate the median and the mean of each run bunch.
```

Comment:
```suggestion
- Read the data and calculate the median and the mean of each run.
```

Note "bunch" should not be used, in my point of view. It sound too "familiar"

Reply:
But we have to do many runs for each test case, so we have to mention it

Reply:
batch ?